src/styles/backupNotification.styl

```diff
@@ -0,0 +1,6 @@
+.backup-notification-wrapper
+    margin-top .5rem
+    margin-right .5rem
+    margin-left .5rem
+    background rgba(255, 255, 255, .16)
```

Comment:
Don't we have semantic var for that?

Reply:
I don't know about this one. It is a workaround because I can not use backdrop-filter.